bootstrap/bootstrap-storage.sh

```diff
@@ -41,60 +40,57 @@ else
   JAVA="${JAVA_HOME}/bin/java"
 fi
 
-TABLE_INITIALIZER_MAIN_CLASS=com.hortonworks.streamline.storage.tool.TablesInitializer
```

Comment:
Why dont we use the TabesInitializer itself to download the library ? Looking for patterns like below can be error prone.